text.go

```diff
@@ -45,6 +45,8 @@ type TextStyle struct {
 	Symbol bool // Use the system symbol font.
 	// Since: 2.1
 	TabWidth int // Width of tabs in spaces
+	// Since 2.5
```

Comment:
Missing colon on this line (after "Since").
It is probably worth adding a line that this is currently only supported in TextGrid so we don't mislead until it is available everywhere.